src/test/java/net/sf/jabref/logic/formatter/bibtexfields/HtmlToLatexFormatterTest.java

```diff
@@ -45,7 +45,7 @@ public void testHTML() {
         assertEquals("{\\\"{a}}", formatter.format("&auml;"));
         assertEquals("{\\\"{a}}", formatter.format("&#228;"));
         assertEquals("{\\\"{a}}", formatter.format("&#xe4;"));
-        assertEquals("{$\\Epsilon$}", formatter.format("&Epsilon;"));
+        assertEquals("{{$\\Epsilon$}}", formatter.format("&Epsilon;"));
```

Comment:
In my opinion the capitalization shouldn't be handled by the HtmlToLatex formatter; we have a special cleanup for preserving capitalization. BibLatex has absolutely no problem with a simple $\Epsilon$. Similarly, I don't see any reason to not write ä as `{\\"a}` (so removing the brackets around the a). Maybe, Biblatex don't even need the outer brackets.